nx/include/switch/services/set.h

```diff
@@ -66,3 +66,6 @@ void setsysExit(void);
 
 /// Gets the current system theme.
 Result setsysGetColorSetId(ColorSetId* out);
+
+/// Gets the serial number.
```

Comment:
Should document the parameters and mention that the output buffer needs to be able to hold at least 0x18 bytes.